app/javascript/utils/statusUtils.js

```diff
@@ -47,11 +47,31 @@ export const LEASE_UP_SUBSTATUS_OPTIONS = {
   ]
 }
 
+export const LEASE_UP_STATUS_VALUES = LEASE_UP_STATUS_OPTIONS.map(option => option.value)
+export const LEASE_UP_SUBSTATUS_VALUES =
+  map(LEASE_UP_SUBSTATUS_OPTIONS, (substatusList) => map(substatusList, 'value'))
+    .flat()
+
 export const getLeaseUpStatusClass = status => {
   const statusOption = find(LEASE_UP_STATUS_OPTIONS, { value: status })
   return statusOption ? statusOption.style : 'tertiary'
 }
 
+export const getStatusPillClass = status => {
+  const statusOption = find(LEASE_UP_STATUS_OPTIONS, { value: status })
+  return statusOption ? statusOption.pillStyle : 'status-pill-no-status'
+}
+
+export const getStatusPillLabel = status => {
+  const statusOption = find(LEASE_UP_STATUS_OPTIONS, { value: status })
+  return statusOption ? statusOption.label : 'Comment'
```

Comment:
This was actually a mistake in the figma design I think. There will always be a status. Adding a comment just creates another field update comment with the same status/substatus and a different comment.

Reply:
I thought this would come up, I'll ask chris about it. We might want it to show as "comment" if multiple history items have the same status in a row.

Reply:
When I brought it up when looking at the designs it sounded like he was fine with just doing what we do now, which is just showing the status and sub-status that's there, but we can re-confirm

Reply:
If you already asked and that's what we show now, that's fine with me! The designs also show duplicate statuses one after the other.